### PR TITLE
fix quay image link in integration details page

### PR DESCRIPTION
**Pull request by karthikjeeyar**
## Fixes 

https://issues.redhat.com/browse/HAC-2811

## Description

After creating an integration test through the UI, if I view its detail page and click on the provided image bundle link, it tries to access that while staying within console and shows me a 404.

## Type of change

- [x] Bugfix

## Screen shots / Gifs for design review 

Before:

Screenshot 2023-01-06 at 3 20 00 PM

After:

https://user-images.githubusercontent.com/9964343/219626083-e16acd59-833d-42ae-8d7a-9ba2d4af09db.mov

## How to test or reproduce?

Add an integration test and visit the integration test details page. (use ?mvp=false flag to enable integration tests tab)

## Browser conformance: 

- [x] Chrome
- [x] Firefox
- [x] Safari
- [ ] Edge

### CODE REVIEW

File: src/components/IntegrationTest/tabs/IntegrationTestOverviewTab.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ const IntegrationTestOverviewTab: React.FC<IntegrationTestOverviewTabProps> = ({
               <DescriptionListGroup>
                 <DescriptionListTerm>Image bundle</DescriptionListTerm>
                 <DescriptionListDescription>
-                  <ExternalLink href={integrationTest.spec.bundle}>
+                  <ExternalLink href={`https://${integrationTest.spec.bundle}`}>
```
Comment (rottencandy) [inline src/components/IntegrationTest/tabs/IntegrationTestOverviewTab.tsx:88]: Looks like it's possible to create integration test with bundle url already having http, should we add a check in that case?

**Review by rottencandy — APPROVED** [review]
/lgtm

**Review by rohitkrai03 — APPROVED** [review]
/lgtm

**Pull request merged**